_docs/setup/install-kubernetes.md

```diff
@@ -14,57 +14,57 @@ Quick Start instructions to install and configure Istio in a Kubernetes cluster.
 
-* Install the Kubernetes client [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/), or upgrade to the latest
-  version supported by your cluster.
+    ```bash
+    oc adm policy add-scc-to-user anyuid -z istio-ingress-service-account -n istio-system
+    oc adm policy add-scc-to-user anyuid -z istio-egress-service-account -n istio-system
+    ```  
 
-* Upgrading Istio is not yet supported. If you previously installed Istio on this cluster, please uninstall first by following the
-  uninstalling steps and yaml files matching your current version.
-  For the {{ site.data.istio.version }} release, the [uninstalling]({{home}}/docs/setup/install-kubernetes.html#uninstalling) steps are located at the end of this page.
+* Install or upgrade the Kubernetes client [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) to match
+  version supported by your cluster (version 1.7 or newer for CRD support).
+
+* If you installed one of our v0.1.x alpha versions before please [uninstall](https://istio.io/v-0.1/docs/tasks/installing-istio.html#uninstalling) it first.
 
 ## Installation steps
 
 Starting with the {{ site.data.istio.version }} release, Istio is installed in its own `istio-system` namespace,
 and can manage micro-services from all other namespaces.
 
-1. Go to the [Istio release](https://github.com/istio/istio/releases) page, to download the installation file corresponding to your OS or run 
+1. Go to the [Istio release](https://github.com/istio/istio/releases) page, to download the installation file corresponding to your OS or run
    ```bash
    curl -L https://git.io/getIstio | sh -
-   ``` 
-   to download and extract the latest release automatically (on MacOS and Ubuntu).
+   ```
+   to download and extract the latest stable release automatically (on MacOS and Ubuntu).
 
-1. Extract the installation file, and change directory to the location where the files were extracted. The following instructions 
+1. Extract the installation file, and change directory to the location where the files were extracted. The following instructions
    are relative to this installation directory.
    The installation directory contains:
-    * yaml installation files for Kubernetes
-    * sample apps
-    * the `istioctl` client binary, needed to inject Envoy as a sidecar proxy, and useful for creating routing rules and policies.
-    * the istio.VERSION configuration file.
+    * Installation `.yaml` files for Kubernetes in `install/`
+    * Sample apps in `samples/`
+    * The `istioctl` client binary in the `bin/` directory. `istioctl` is used when manually injecting Envoy as a sidecar proxy, and useful for creating routing rules and policies.
+    * The `istio.VERSION` configuration file.
 
-1. Add the `istioctl` client to your PATH if you download the installation file from [Istio release](https://github.com/istio/istio/releases).
```

Comment:
red deleted part (if ...) was noise (as the script doesn't set the path, just suggest you to do it, as I had tried to explain when that was added)